scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -195,6 +195,13 @@ abstract class FileSource extends SchemedSource with LocalSourceOverride {
             "[" + this.toString + "] No good paths in: " + hdfsPaths.toString)
         }
       }
+
+      case Local(_) => {
```

Comment:
I guess not. The previous (perhaps questionable) logic is that strict means something is there, but maybe not all the paths. Nothing being present still won't run. Since currently Local only supports one path (even though it could support more), I suppose this is correct.